src/resources/Pipelines/Statements/Statements.ts

```diff
@@ -92,4 +93,15 @@ export default class Statements extends Resource {
             })
         );
     }
+
+    bulkGet(pipelineId: string, options: BulkGetStatementsParams) {
+        const {ids, ...allQueryStringOptions} = options;
```

Comment:
[optional] deconstruct directly in the params?

```suggestion
    bulkGet(pipelineId: string, {ids, ...allQueryStringOptions}: BulkGetStatementsParams) {
```